### PR TITLE
Add asset prompt for rating/simplification subsets

### PR DESCRIPTION
**Pull request by thomasw21**

### CODE REVIEW

File: templates/asset/ratings/templates.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+      \n{{questions[aspect]}} \n\n|||\n\n{% if rating > 50 %}\n    Yes\n{% else %}\n\
+      \    No\n{% endif %}"
+    name: asset_ratings1
+    reference: Taking questions from the original paper, we use rating to establish
+      a binary classification problem.
+    task_template: false
+  d2bed959-29ab-4962-a106-dc91c00f3f03: !Template
+    id: d2bed959-29ab-4962-a106-dc91c00f3f03
+    jinja: "{% set statements= [ \"the second sentence expresses the underlying meaning\
+      \ the best.\",  \"the second sentence if more fluent.\", \"the second sentence\
```
Comment (VictorSanh) [inline templates/asset/ratings/templates.yaml:18]: **is** more fluent

File: templates/asset/ratings/templates.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+      \ is easier to read and understand.\"] %}\n\nFirst sentence: {{original}}\n\n\
+      Second sentence: {{simplification}}\n\nRate the following statement from 0 (strongly\
+      \ disagree) to 100 (strongly agree): {{statements[aspect]}} \n\n|||\n\n{{rating}}"
+    name: asset_ratings2
+    reference: Require the model to output the rating
+    task_template: false
```
Comment (VictorSanh) [inline templates/asset/ratings/templates.yaml:24]: this template looks like the original task to me right? if so, it should be `true`

File: templates/asset/simplification/templates.yaml [hunk 3]
```diff
@@ -0,0 +1,34 @@
+dataset: asset
+subset: simplification
+templates:
+  0f0e55f9-28b4-4844-b65d-b9544a0918eb: !Template
+    id: 0f0e55f9-28b4-4844-b65d-b9544a0918eb
+    jinja: "{% set random_simplification_id = range(0, simplifications | length) |\
+      \ choice %}\n{{original}}\n\nHow would I say this in another way? \n\n|||\n\n\
+      {{simplifications[random_simplification_id]}}"
```
Comment (VictorSanh) [inline templates/asset/simplification/templates.yaml:8]: let's simplify the jinja:
```jinja
{{original}}
How would I say this in another way?|||
{{simplifications | choice}}
```

File: templates/asset/simplification/templates.yaml [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+    name: asset_simplification1
+    reference: Rewrite text using one random simplification
+    task_template: false
```
Comment (VictorSanh) [inline templates/asset/simplification/templates.yaml:11]: looks like the origianl task to me

File: templates/asset/simplification/templates.yaml [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+  3cbfbc1c-6876-4dd7-b7db-45fb3233a667: !Template
+    id: 3cbfbc1c-6876-4dd7-b7db-45fb3233a667
+    jinja: "{% set random_simplification_id = range(0, simplifications | length) |\
+      \ choice %}\n{{simplifications[random_simplification_id]}}\n\nHow would I say\
+      \ this in another way? \n\n|||\n\n{{original}}"
```
Comment (VictorSanh) [inline templates/asset/simplification/templates.yaml:16]: same comment on the jinja

File: templates/asset/simplification/templates.yaml [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+      I''d like to explain to my child "{{original}}". How would I do so?
+
+
+      |||
+
+
+      {{simplifications[random_simplification_id]}}'
+    name: asset_simplification3
+    reference: Implicit simplification request
+    task_template: false
```
Comment (VictorSanh) [inline templates/asset/simplification/templates.yaml:34]: true task

File: templates/asset/simplification/templates.yaml (hunk 5 above)
Comment (VictorSanh) [reply templates/asset/simplification/templates.yaml:16]: same for `asset_simplification3`

**Review by VictorSanh — CHANGES_REQUESTED** [review]
Thanks @thomasw21 !

I kind of feel like the "simplified" versions don't look that much simplified so "say it differently" looks good to me!

Left a few comments
Ping me when you have addressed them, will merge then!

**Comment by VictorSanh** [comment]
A funky one that you can add for ratings is:

```jinja
First sentence: {{original}}
Second sentence: {{simplification}}
I am scoring these simplification exercises. How easier to read is the second sentence on a scale from 0 (harder to read) to 100 (easier to read)?|||
{{rating}}
```

**Comment by thomasw21** [comment]
@VictorSanh I've addressed your comments, and added your template as per your suggestion (though I'm not exactly sure how different the prompt is from `asset_ratings2`).

**Review by VictorSanh — APPROVED** [review]
merging now, please update the spreadsheet!

**Pull request merged**